scripts/installer/cloudstack-help-text

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+printf "\n"
+printf "Thank you for installing Apache CloudStack !\n"
+printf "\n"
+printf " * Project:          https://cloudstack.apache.org/\n"
+printf " * Documentation:    https://docs.cloudstack.apache.org/\n"
```

Comment:
After this line, should we add the link to release notes of the current release/version, for example:
```
http://docs.cloudstack.apache.org/en/${$ACS_VERSION:-"latest"}/releasenotes/index.html
```

Reply:
@rohityadavcloud good idea. I will add it